portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,261 @@
+
+                    </p>
+                    <p>
+                         I wish to travel all around the world one day and develop technologies to help everyone.
+                    </p>
+                </div>
+            </div>
+            <div class="col-12 col-lg-5 about-box">
+                <div class="about-box-img">
+                    <img src="Images/IMG_5111(1).jpg" alt="about" data-at2x="Images/IMG_5111(1).jpg" height="300px" width="400px">
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<br>
+<!-- project -->
+
+<div id="projects" class="portfolio-container section-container">
+    <div class="container">
+        <div class="row">
+            <div class="col portfolio section-description">
+                <h2>My Projects</h2>
+                <div class="divider-1"><span></span></div>
+                <p>My 3 recent projects. Check them out!</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-4 portfolio-box">
+                <div class="portfolio-box-image">
+                    <img src="Images/12.jpg" alt="" data-at2x="Images/12.jpg" height="200px" width="300px">
+                </div>
+                <h3>Jarvis: Real time gun detection  </h3>
+                <div class="portfolio-box-date">April 2020</div>
+                <p>Developed a YOLO architecture based weapon detection system to equip surveillance cameras to detect offensive weapons and send
+                real-time location and image results to the police through an application</p>
+            </div>
+            <div class="col-md-4 portfolio-box wow fadeInDown">
+                <div class="portfolio-box-image">
+                    <img src="Images/voice_controlled_smart_home.jpg" alt="" data-at2x="Images/voice_controlled_smart_home.jpg" height="200px" width="200px">
+                </div>
+                <h3>VMouse : Voice controlled mouse</h3>
+                <div class="portfolio-box-date">May 2020</div>
+                <p>Developed a prototype bonsai based application of a voice based digital assistant to accept voice commands and perform GUI based actions</p>
+            </div>
+            <div class="col-md-4 portfolio-box wow fadeInUp">
+                <div class="portfolio-box-image">
+                    <img src="Images/data_quality.png" alt="" data-at2x="Images/data_quality.png" height="200px" width="200px">
+                </div>
+                <h3>Data Quality and Privacy analyser </h3>
+                <div class="portfolio-box-date">January 2020</div>
+                <p>Devised a platform to determine the reliability and quality of a dataset for user application.</p>
+            </div>
+        </div>
+        </div>
+</div>
+
+<!-- photo gallery -->
+<div id="photos" class="image-container">
+  <h2><center>Few Memories</center></h2>  
+  <div id="myCarousel" class="carousel slide" data-ride="carousel">
```

Comment:
about the `carousel ` height that we discussed in the meeting. I was curious if you tried to play with the `max-height` CSS property? I was thinking just changing the `height` property may not help if there is a default `max-height` set by default.